packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -448,7 +448,7 @@ describe('Validating rule metadata', () => {
     describe(ruleName, () => {
       it('`name` field in rule must match the filename', () => {
         // validate if rule name is same as url
-        // there is no way to access this field but its used only in generation of docs url
+        // there is no way to access this field but it's used only in generation of docs url
```

Comment:
hmm
should be in this PR?

Reply:
A previous version of this changeset included some actual code changes here, and this was leftover after reverting those haha. I can remove it 👍